helm/openwhisk/values-metadata.yaml

```diff
@@ -1560,4 +1560,24 @@ metrics:
       description: "true to enable user metrics"
       type: "boolean"
       required: true
+pdb:
+  _metadata:
+  label: "Pod's disruption budget settings"
+  description: "Pod Disruption settings for voluntary and nonvoluntary disruptions"
+  enable:
+    _metadata:
+      label: "Enable pod disruption budget for deployment"
+      description: "true to enable pod disruption budgets for pods"
+      type: "boolean"
+      required: true
+  component:
```

Comment:
The tool that processes values-metadata.yaml won't understand the pattern you are describing here by using `component`.   I think the choices are either (a) replicate the component name & maxUnavailable stanza for each component (zookeeper, controller, etc.) that you have defined in values.yaml or (b) leave off the entries in values-metadata.yaml.    I'm fine with either option.

Reply:
Sure will do that. Also, will do a similar modification for prob settings as well.